commit afa80cb2b457bf8d64f775bed307588610476c41
Author: Michael R Sweet <michaelrsweet@gmail.com>
Date:   Tue Jan 3 13:52:47 2017 -0500

    Don't treat "localhost.localdomain" as an allowed replacement for localhost, since it isn't.

diff --git a/scheduler/client.c b/scheduler/client.c
index 42010def1..20ccf11a9 100644
--- a/scheduler/client.c
+++ b/scheduler/client.c
@@ -3890,9 +3890,6 @@ valid_host(cupsd_client_t *con)		/* I - Client connection */
 
     return (!_cups_strcasecmp(con->clientname, "localhost") ||
 	    !_cups_strcasecmp(con->clientname, "localhost.") ||
-#ifdef __linux
-	    !_cups_strcasecmp(con->clientname, "localhost.localdomain") ||
-#endif /* __linux */
             !strcmp(con->clientname, "127.0.0.1") ||
 	    !strcmp(con->clientname, "[::1]"));
   }
